themes/custom/az_barrio/az_barrio.info.yml

```diff
@@ -3,8 +3,6 @@ type: theme
 description: 'Basic structure for a Bootstrap Barrio SubTheme.'
 core_version_requirement: ^8.8 || ^9
 base theme: bootstrap_barrio
-logo: images/logo.png
-footer_logo: images/logo.png
```

Comment:
Should we remove the actual PNG image file from the repo as well?
https://github.com/az-digital/az_quickstart/blob/issue/516-remaining-logo-issues/themes/custom/az_barrio/images/logo.png

Reply:
We still use the logo here: https://github.com/az-digital/az_quickstart/blob/main/themes/custom/az_barrio/config/install/az_barrio.settings.yml#L68